src/WTG.Analyzers.Utils/DirectiveHelper.cs

```diff
@@ -30,6 +31,16 @@ public static bool BoundingTriviaSplitsNodes(SyntaxTrivia startTrivia, SyntaxTri
 				endToken = endToken.GetPreviousToken();
 			}
 
+			if (startToken.IsKind(SyntaxKind.CommaToken))
```

Comment:
what happens if startToken and endToken are the same comma?

```csharp
		enum ConditionalEnum
		{
			A
#if true
			,
#endif
			B
		}
```

Reply:
surprisingly, its happy with that, though the test I needed for it was such convoluted syntax that I really didn't want to commit it to anything 😄 

if you insist though I can add it... it was basically `Foo(1 #if true , #endif -1)`

Reply:
Whilst it is definitely not something that would happen in real code, I can see it easily happening as an intermediate state while editing (eg. moving lines around with Alt + Shift + Up/Down). I can see the test having value in the sense that we are asserting it doesn't crash or produce anything confusing, rather than asserting that it produces the correct error. I consider it to be the same category as SplitListElementsNewLineComma.